### PR TITLE
Fix auto-painting tile/wall being rejected from Hand of Creation

### PR DESCRIPTION
**Pull request by csmir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
   * If there is no section called "Upcoming changes" below this line, please add one with `## Upcoming changes` as the first line, and then a bulleted item directly after with the first change.
 
 ## Upcoming changes
+* Fixed painting wall/tile being rejected from hand of creation. (@Rozen4334)
```
Comment (hakusaro) [inline CHANGELOG.md]: If you could put this at the bottom of the changelog that'd be posh (some of the changes below yours happened approximately 6-9 months ago)

**Review by hakusaro — DISMISSED** [review]
Other than minor changelog nits, this is great! thank you so much!

lgtm_trick

**Review by drunderscore — DISMISSED** [review]
Seems totally fine. Looks like I myself previously forgot to handle the case for painting walls with these items when I last touched this, so thanks for the fix :heart:

**Comment by hakusaro** [comment]
@Rozen4334 fyi we don't have the ability to modify or edit this PR so we can't update it/merge with base branch; that's the minimum to merge

**Comment by csmir** [comment]
Should be good for review now

**Review by hakusaro — APPROVED** [review]
lgtm!

**Review by QuiCM — APPROVED** [review]

**Review by drunderscore — APPROVED** [review]
LGTM!

**Pull request merged**